synthtool/gcp/templates/python_library/docs/conf.py.j2

```diff
@@ -346,12 +346,12 @@ texinfo_documents = [
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
     "python": ("http://python.readthedocs.org/en/latest/", None),
```

Comment:
Python is also redirecting to the https endpoint.

Reply:
Thanks Tim!